### ISSUE TITLE
flake8-docstring does not detect docstrings written in a non-Google style

### ISSUE THREAD

**Issue by nbro10** [PyCQA/flake8-docstrings#60]
We're using the Google's docstring style in our project. I've installed this package in order to make sure that my docstrings are consistent across my project's files. However, it seems that it is not able to detect that one of the modules is not using the Google's docstring style.

Here are my dependencies

```
...
mccabe                        0.6.1      McCabe checker, plugin for flake8
...
flake8                        3.9.0      the modular source code checker: pep8 pyflakes and co
flake8-docstrings             1.7.0      Extension for flake8 which uses pydocstyle to check docstrings
...
pydocstyle                    6.3.0      Python docstring style checker
pyflakes                      2.3.1      passive checker of Python programs
```

Let me know if you need more dependencies. I'm using Python 3.8.13.

I'm running flake8 as follows (but you can probably ignore poetry - just in case, I'm using poetry 1.4.0)

```
poetry run flake8 myfile.py
```

or (it does not matter - same result)

```
poetry run flake8 myfile.py --docstring-convention google
```

Here are my flake8 configs in the `tox.ini`

```
[flake8]
max-line-length = 100
max-complexity = 12
docstring-convention = google
max-doc-length = 100
ignore=D100,D101,D102,D103,D104,D105,D107,D200,D202,D205,D212,D415,E203,W503
```

So, for completeness, in `myfile.py` I have a function like this (which does not use Google's docstring style)

```
def my_func(a):
    """
    Do something.

    :param a: Nice.
    :return: Whatever.
    """
```

So, does this tool actually detect these issues, or did I misinterpret the purpose of this tool?

The README says that pydocstyle 4.0.0 added support for the Google's style, so I'd expect this to be detected, given my dependencies

**Comment by sigmavirus24** [comment]
This tool doesn't implement the checks but instead plugs pydocstyle into flake8. Pydocstyle is a separate project

**Comment by sigmavirus24** [comment]
I'm fairly certain choosing a convention doesn't forbid another.
